cubi_tk/sodar/download_sheet.py

```diff
@@ -146,6 +146,21 @@ def execute(self) -> typing.Optional[int]:
         return 0
 
     def _write_file(self, out_path, file_name, text):
+        """Write file.
+
+        :param out_path: Path to output directory.
+        :type out_path: str
+
+        :param file_name: File name as provided in SODAR. If input includes more than file name, extra information
+        is removed. File should be created in output path root.
+        Example: 'PROJECT/i_Investigation.txt' -> 'i_Investigation.txt'.
+        :type file_name: str
+
+        :param text: Text to be written in file.
+        :type text: str
+        """
+        # Remove extra info - use basename only
+        file_name = file_name.split('/')[-1]
```

Comment:
Is this always looking like a (posix?) path?

If yes, that's ok, otherwise we should probably use pathlib.

Reply:
So far I've seen only two options: `PROJECT/i_Investigation.txt` and `i_Investigation.txt`.
From the brief that I talk to Mikko and the little change this part went over the years, I don't expect much change.